docs/BrandColors.stories.tsx

```diff
@@ -18,7 +22,64 @@ export default meta;
 
 type Story = StoryObj<typeof ColorSwatchGroup>;
 
-export const DefaultStory: Story = {
+export const Figma: Story = {
   render: () => <ColorSwatchGroup swatchData={tokens.global.brandColors} />,
-  name: 'Default',
+};
+
+export const CSS: Story = {
+  render: () => {
+    const [brandColors, setBrandColors] = useState<Color>({});
+
+    useEffect(() => {
+      // Getting all CSS variables that start with '--brand-colors' from all stylesheets
```

Comment:
perhaps this can be extracted out to a utils as well? getBrandColorsFromStylesheet?

Reply:
Great suggestion! Managed to refactor into one function and removed 60 lines of code! Updated here https://github.com/MetaMask/design-tokens/pull/594/commits/a6a6567261bb879ec6e20e5d6445165939717c46